packages/block-editor/src/components/letter-spacing-control/style.scss

```diff
@@ -0,0 +1,6 @@
+// Needs specificity to override the very specific generated View component CSS.
+.block-editor-letter-spacing-control .components-flex-item > .components-text.components-input-control__label {
+	padding-bottom: 0;
```

Comment:
I wonder whether the [component base label styles](https://github.com/WordPress/gutenberg/blob/92662feeb220336fb28f8b79869cb251a7db4e27/packages/components/src/input-control/styles/input-control-styles.tsx#L242) themselves should be the "source of truth".

That is, whether all labels fall in line with the definitions in the base component or the base component itself bumps its margin. 

I'm basing this question on the premise that the use of base component controls and labels will be more widespread in the future (?)

Reply:
I was a bit slow in spotting your comment here sorry. I created an [alternate PR](https://github.com/WordPress/gutenberg/pull/36387) standardizing the input control labels line-height and margins when used within the `ToolsPanel`. I did it there so as to not mess with spacing if the control was used elsewhere.

The change of label text here to align with the typography control mockups still needs doing either way.

Reply:
Ultimately, I think having a single source of truth for the label styles in the sidebar should be the goal. I'd say we'll need to take incremental steps to get there. That might include needing to update all the components to use the new Emotion/context style system.

Reply:
Ah great, thanks @aaronrobertshaw I'll roll my changes back in this PR. Your changes make a lot more sense.